packages/ui/src/components/molecules/LeanBox.tsx

```diff
@@ -119,17 +153,29 @@ export const LeanBox: FC<LeanBoxProps> = ({
         )}
         {tag && <Tag>{tag}</Tag>}
         <RightContent>
+          {throbber && <Throbber size={15} strokeWidth={2} />}
           {rightText && (
-            <Typography variant={rightTextVariant} color={rightTextColor}>
+            <Typography
+              variant={rightTextVariant}
+              color={rightTextColor}
+              $fontSize={14}
+              $fontWeight="normal"
+            >
               {rightText}
             </Typography>
           )}
-          {rightImage && <ImageContainer>{rightImage}</ImageContainer>}
+          {rightImage && (
+            <ImageContainer>
+              <Image src={rightImage as string} alt="image" />
+            </ImageContainer>
+          )}
```

Comment:
leftImage and rightImage is a reactNode, It should not be rendered this way.

Reply:
done